parachain/pallets/inbound-queue/src/lib.rs

```diff
@@ -91,6 +88,9 @@ pub mod pallet {
 
 		type WeightInfo: WeightInfo;
 
+		// Gateway contract address
+		type GatewayAddress: Get<H160>;
```

Comment:
```suggestion
		#[pallet::constant]
		type GatewayAddress: Get<H160>;
```

Reply:
Resolved in [6693a52](https://github.com/Snowfork/snowbridge/pull/953/commits/6693a52a450f830726e8cd6b129b0702517e0cfb)